exact_multiple_string_matching/fast_practical_multi_string_matching.py

```diff
@@ -0,0 +1,151 @@
+class AhoCorasickNode:
```

Comment:
A general question: what prevented you from using the other Aho-Corasick automaton?